src/Utils/obiDialogParser.ts

```diff
@@ -11,7 +11,8 @@ import * as stripJsonComments from 'strip-json-comments'
 
 enum OBIStepType {
     BEGIN_DIALOG = "Microsoft.BeginDialog",
-    END_TURN = "Microsoft.EndTurn",
+    END_DIALOG = "Microsoft.EndDialog",
+    END_TURN = "Microsoft.EndTurn",  // TODO(thpar) : Obsolete / delete ?
```

Comment:
I believe this is still a valid OBIStep type.  We just don't see it in any of the exported dialogs from CCI

Reply:
Got it.  Removed comment.